phpBB/docs/INSTALL.html

```diff
@@ -147,7 +147,7 @@ <h1>Install</h1>
 			<li>Oracle</li>
 		</ul>
 		</li>
-		<li><strong>PHP 7.1.0+</strong> up to and including <strong>PHP 7.4</strong> with support for the database you intend to use.</li>
+		<li><strong>PHP 7.1.3+</strong> up to and including <strong>PHP 7.4</strong> with support for the database you intend to use.</li>
```

Comment:
Confirmed minimum PHP 7.1.3?

Reply:
Yes, in line with the symfony requirements

Reply:
Thanks. :+1: